src/infra/storage/local.rs

```diff
@@ -83,16 +83,12 @@ mod test_util {
         let file_text = "Some text";
         let file_name = "a/b/c/new_file.parquet";
 
-        let resp = local.put(file_name, bytes::Bytes::from(file_text)).await;
-        assert!(resp.is_ok());
-
-        let resp = local.get(file_name).await;
-        assert_eq!(resp.unwrap(), bytes::Bytes::from(file_text));
+        local.put(file_name, bytes::Bytes::from(file_text)).await.unwrap();
```

Comment:
We should assert the result, maybe put got error?

Reply:
`res.unwrap()` is semantically equivalent to `assert!(res.is_ok())` — both expressions panic if `res.is_err()`.

```rust
pub const OK: Result<(), ()> = Ok(());
pub const ERR: Result<(), ()> = Err(());

#[test]
fn test_ok() {
    assert!(OK.is_ok());
    OK.unwrap();
}

#[test]
fn test_err1() {
    assert!(ERR.is_err());
}

#[test]
#[should_panic]
fn test_err2() {
    assert!(ERR.is_ok());
}

#[test]
#[should_panic]
fn test_err3() {
    ERR.unwrap();
}
```

[Playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=825cee8a6b7c6ca5a6a2204f6205ef55)